### PR TITLE
Update docs to include SIG-CI

### PR DESCRIPTION
**Pull request by radhikaj**
Signed-off-by: Radhika Jandhyala <radhikaj@microsoft.com>

### CODE REVIEW

File: docs/Community/sig-ci/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+CI/CD Special Interest Group
```
Comment (dthaler) [inline docs/Community/sig-ci/README.md:1]: Please expand CI/CD acronym on first use

File: docs/Community/sig-ci/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+========================================
+
+This SIG is responsible for infrastructure and automation related to CI/CD for the Open Enclave SDK project.
+
+This SIG may also weigh in on specific PRs to ensure that that it is possible to build CI to automate testing the changes suggested in the PR.
```
Comment (dthaler) [inline docs/Community/sig-ci/README.md:6]:
```suggestion
This SIG may also weigh in on specific PRs to ensure that it is possible to build CI to automate testing the changes suggested in the PR.
```

**Review by dthaler — CHANGES_REQUESTED** [review]
please update sig-architecture/README.md as discussed in email

File: docs/Community/sig-ci/README.md (hunk 1 above)
Comment (radhikaj) [reply docs/Community/sig-ci/README.md]: Updated

File: docs/Community/sig-architecture/OWNERS [hunk 3]
```diff
@@ -14,6 +14,7 @@ reviewers:
   - sig-release-chair
   - sig-sgx-chair
   - sig-optee-chair
+  - sig-ci
```
Comment (mingweishih) [inline docs/Community/sig-architecture/OWNERS:17]: Just curious, why this is not sig-ci-chair?

Comment (radhikaj) [reply docs/Community/sig-architecture/OWNERS:17]: It should be, thanks!

File: docs/Community/sig-ci/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+Continuous Integration Continuous Deployment(CI/CD) Special Interest Group
```
Comment (dthaler) [inline docs/Community/sig-ci/README.md:1]:
```suggestion
Continuous Integration/Continuous Deployment (CI/CD) Special Interest Group
```

File: docs/Community/sig-ci/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+Meetings
+--------
+
+No regular meetings are scheduled.
+
+SIG Chair
+---------
+
+Brett Mclaren ([brmclaren](https://github.com/brmclaren))
```
Comment (dthaler) [inline docs/Community/sig-ci/README.md:16]:
```suggestion
Brett McLaren ([brmclaren](https://github.com/brmclaren))
```

**Review by dthaler — APPROVED** [review]

File: docs/Community/sig-ci/OWNERS [hunk 6]
```diff
@@ -0,0 +1,10 @@
+chair:
+  - brmclaren
+
+approvers:
+  - anakrish
+  - codemonkeyleet
+  - mingweishih
+  - rs--
+
+reviewers:
```
Comment (CodeMonkeyLeet) [inline docs/Community/sig-ci/OWNERS:10]: Nit: add newline at EOF

File: docs/Community/sig-ci/README.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
+Continuous Integration/Continuous Deployment (CI/CD) Special Interest Group
+==========================================================================
+
+This SIG is responsible for infrastructure and automation related to CI/CD for the Open Enclave SDK project.
+
+This SIG may also weigh in on specific PRs to ensure that it is possible to build CI/CD to automate testing the changes suggested in the PR.
```
Comment (CodeMonkeyLeet) [inline docs/Community/sig-ci/README.md:6]: This is probably up to the SIG chair to amend as needed, but I imagine that for the CD part of this SIG, they will also likely have input related to matters of release and platform targeting.

**Review by CodeMonkeyLeet — APPROVED** [review]

File: docs/Community/sig-ci/README.md (hunk 7 above)
Comment (radhikaj) [reply docs/Community/sig-ci/README.md]: Added text that effect.

**Pull request merged**